components/script/dom/html/htmlmapelement.rs

```diff
@@ -53,3 +58,23 @@ impl HTMLMapElement {
             .collect()
     }
 }
+
+impl HTMLMapElementMethods<crate::DomTypeHolder> for HTMLMapElement {
+    // <https://html.spec.whatwg.org/multipage/#dom-map-name>
+    make_getter!(Name, "name");
```

Comment:
This is currently unused. Are you planning to also implement `usemap` where this is referenced? https://html.spec.whatwg.org/multipage/image-maps.html#attr-hyperlink-usemap